### ISSUE TITLE
NoSwing flagging for AutoClicker with 0VL

### ISSUE THREAD

**Issue by heine789** [funkemunky/Fiona-Issues#4]
I know there is a format but this was on your test server so you probably know more about the plugins and versions you have on there etc

https://www.youtube.com/watch?v=GCnXfwVKPXc&feature=youtu.be

I basically just turned on NoSwing and it started flagging me for AutoClicker with 0VL every 1-2 seconds

**Comment by funkemunky** [comment]
I don't believe that has anything to do with Fiona, but rather your client's NoSwing module being a bit broken. Seeing as it only starts flagging when you turn the module on, it might as well just be an unexpected detection.
